### ISSUE TITLE
Android login error: Something went wrong :(   - cannot login to News with Android App

### ISSUE THREAD

**Issue by wgregori** [owncloud/news#1013]
I'm running Nextcloud 9.0.50 (stable).  I have tried to login to News with two different android devices and receive the message Error: Something went wrong  -- 

Additionally, I successfully used the Import Account utility to capture the correct URL.

Thanks for the work on this very important app I'm looking forward to using it on my portable devices.

Wayne

**Comment by Gerii** [comment]
You're wrong here. Either try in the Nextcloud/news repository or in the repo of the appropriate app.
